### PR TITLE
Ftl names

### PR DESCRIPTION
**Pull request by OCOtheOmega**
# Description

Add name and description to locale and in files.

# TODO

# Media

# Changelog

no cl no fun -death

### CODE REVIEW

**Comment by DEATHB4DEFEAT** [comment]
changelog unnecessary

**Review by DEATHB4DEFEAT — APPROVED** [review]

File: Resources/Locale/en-US/SimpleStation14/Prototypes/Uplink/uplink-catalog.ftl [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+uplink-book-fireball-desc = Throw a massive explosive at people, don't get caught in it yourself.
+
+uplink-book-heal-name = Healing book
+uplink-book-heal-desc = Target a being and heal them of all their wounds instantly! (WARNING - May be considered necromany if used on the dead)
+
+# Wands
+
+# Armor
+uplink-armor-hardsuit-name = Hardsuit
+uplink-armor-hardsuit-desc = A someone damage resistant, but heavy hardsuit that allows eva and protection in fights. Also acts as wizard robes, allowing you to use spells.
```
Comment (Pspritechologist) [inline Resources/Locale/en-US/SimpleStation14/Prototypes/Uplink/uplink-catalog.ftl]:
```suggestion
uplink-armor-hardsuit-desc = A somewhat damage resistant, but heavy hardsuit that allows eva and protection in fights. Also acts as wizard robes, allowing you to use spells.
```

File: Resources/Locale/en-US/job/job-description.ftl [hunk 2]
```diff
@@ -41,3 +41,5 @@ job-description-security = Catch criminals and enemies of the station, enforce t
 job-description-serviceworker = Learn the basics of bartending, cooking, and growing plants.
 job-description-warden = Patrol the security department, ensure that no one is stealing from the armory, and make sure that all prisoners are processed and let out when their time is up.
 job-description-zookeeper = Put on a joyful display of cute animals and space carps for all the crew to see. Not available on all maps!
+job-description-sai = Follow your laws and serve the humans to the best of your ability.
+job-description-css = As chief supervisor, your main duties would include managing the crew, ensuring that everyone is performing their duties correctly.
```
Comment (Pspritechologist) [inline Resources/Locale/en-US/job/job-description.ftl]: There's another PR open for this already, but they have not been responsive.
Take a look at that one to see what the descriptions are, and where the file is.

File: Resources/Locale/en-US/job/job-names.ftl [hunk 3]
```diff
@@ -42,7 +42,7 @@ job-name-boxer = boxer
 job-name-zookeeper = zookeeper
 
 
-#Nyano
+# Nyano
```
Comment (Pspritechologist) [inline Resources/Locale/en-US/job/job-names.ftl:45]: Yes please, thank you.

File: Resources/Locale/en-US/job/job-names.ftl [hunk 4]
```diff
@@ -60,6 +60,10 @@ job-name-paramedic = paramedic
 job-name-guard = prison guard
 job-name-martialartist =  martial artist
 
+# Park
+job-name-sai = station AI
+job-name-css = chief service supervisor
+
```
Comment (Pspritechologist) [inline Resources/Locale/en-US/job/job-names.ftl]: These should go in the SimpleStation FTL folder. 
Should be Resources/Locale/en-US/SimpleStation14/  if I'm not mistaken.

File: Resources/Locale/en-US/job/job-names.ftl [hunk 5]
```diff
@@ -69,6 +73,8 @@ JobCargoTechnician = cargo technician
 JobCentralCommandOfficial = central command official
 JobChaplain = chaplain
 JobChef = chef
+JobSAI = station AI
+JobChiefServiceSupervisor = chief service supervisor
```
Comment (Pspritechologist) [inline Resources/Locale/en-US/job/job-names.ftl]: These should go in the SimpleStation FTL folder.
Should be Resources/Locale/en-US/SimpleStation14/ if I'm not mistaken.

File: Resources/Locale/en-US/prototypes/roles/antags.ftl [hunk 6]
```diff
@@ -32,4 +32,7 @@ roles-antag-evil-twin-description = Replace your twin.
 roles-antag-minor-antag-name = Minor Antag
 roles-antag-minor-antag-objective = Make your way through a normal shift, considering your ulterior motives.
 
+roles-antag-wizard-antag-name = Wizard
+roles-antag-wizard-objective = The Wizard Federation has selected you to eliminate the crew.
+
```
Comment (Pspritechologist) [inline Resources/Locale/en-US/prototypes/roles/antags.ftl]: These should go in the SimpleStation FTL folder.
Should be Resources/Locale/en-US/SimpleStation14/ if I'm not mistaken.

File: Resources/Locale/en-US/store/categories.ftl [hunk 7]
```diff
@@ -15,3 +15,7 @@ store-category-pointless = Pointless
 
 # Revenant
 store-category-abilities = Abilities
+
+# Wizard
+store-category-books = Books
+store-category-wands = Wands
```
Comment (Pspritechologist) [inline Resources/Locale/en-US/store/categories.ftl:21]: These should go in the SimpleStation FTL folder.
Should be Resources/Locale/en-US/SimpleStation14/ if I'm not mistaken.

File: Resources/Prototypes/SimpleStation14/Traits/abilities.yml [hunk 8]
```diff
@@ -1,7 +1,8 @@
 - type: trait
   id: DragonsBlood
-  name: Draconic Bloodstream
-  description: You have some of the great Space Dragon's healing abilities, and have Ichor in your blood.  REQUIRES Reptilian species
+  name: trait-dragons-blood-name
+  description: trait-dragons-blood-description
+  REQUIRES Reptilian species
```
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Traits/abilities.yml]: This looks like an error to me.

**Review by Pspritechologist — CHANGES_REQUESTED** [review]
Good changes overall. Sorry you have to clean up our sloppy work :P

File: Resources/Locale/en-US/store/categories.ftl (hunk 7 above)
Comment (OCOtheOmega) [reply Resources/Locale/en-US/store/categories.ftl:21]: better to make categories inside one file

**Review by OCOtheOmega — COMMENTED** [review]
i done

**Comment by Pspritechologist** [comment]
I don't see where the trait FTL strings are in the files, such as trait-monochromacy-name, or trait-mute-description. Are they already elsewhere, or am I just missing them?

Death and I had intended to keep any FTL strings for content we've added in the SimpleStation FTL folder for ease of editing it. I understand this may not be as useful for someone translating. Since translation is the whole point of it, I will speak to him about changing it.

**Comment by OCOtheOmega** [comment]
> I don't see where the trait FTL strings are in the files, such as trait-monochromacy-name, or trait-mute-description. Are they already elsewhere, or am I just missing them?
> 
> Death and I had intended to keep any FTL strings for content we've added in the SimpleStation FTL folder for ease of editing it. I understand this may not be as useful for someone translating. Since translation is the whole point of it, I will speak to him about changing it.

There others trait death added but in one commit you remove they, soo i just back it.
https://github.com/Devianartaro/EbenGrad/blob/ftl-names/Resources/Locale/en-US/SimpleStation14/Prototypes/Traits/disabilities.ftl

**Review by Pspritechologist — APPROVED** [review]

**Review by DEATHB4DEFEAT — APPROVED** [review]

**Pull request merged**